### DISCUSSION TITLE
deploy_js.sh error: required entry point "fmltc.Admin" never provided

### DISCUSSION THREAD

**Discussion by BalaKasi** [FIRST-Tech-Challenge/fmltc#228] [Q&A]
In the [Deploy javascript code] step, deploy_js.sh raises this error: required entry point "fmltc.Admin" never provided.  I tried providing the full path where the Admin.js file is located - but this didn't resolve.   Please clarify if this is resolvable.

**Comment by lizlooney** [comment]
What is the current directory when you run the deploy_js.sh script?

**Reply by BalaKasi** [reply]
I am in \fmltc\closure-compiler.

**Reply by cmacfarl** [reply]
If you have gradle installed on your machine you can build the closure file via ```./gradlew compileJavascript``` from the root of the project.  If you go that route then gradle handles the closure compiler dependency automagically.  See build.gradle for the implementation.

**Comment by lizlooney** [comment]
The current directory should be fmltc for all of the deploy scripts.
Also, closure-compiler should be in a directory that is alongside (not within) the fmltc directory.
The directory structure should be
+ parent
  + fmltc
    + doc
    + scripts
    + server
      + src
        + js
      + static
        + css
        + training
      + templates
    + terraform
  + closure-compiler
  + closure-library

**Reply by BalaKasi** [reply]
Thanks!  I tweaked the scripts and only the last script - deploy_gae.sh, ended with an error.
